.github/workflows/backup_production_database.yml

```diff
@@ -1,18 +1,87 @@
-name: Production DB nightly backup
+name: Backup database to Azure storage
+
 on:
   workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment to backup
+        required: true
+        default: staging
+        type: choice
+        options:
+          - staging
+          - production
+      backup-file:
+        description: |
+          Backup file name (without extension). Default is ecf_[env]_adhoc_YYYY-MM-DD. Set it explicitly when backing up a point-in-time (PTR) server. (Optional)
+        required: false
+        type: string
+        default: default
+      db-server:
+        description: |
+          Name of the database server. Default is the live server. When backing up a point-in-time (PTR) server, use the full name of the PTR server. (Optional)
+
   schedule:
-    - cron: "30 3 * * *" # 03:30 UTC
+    - cron: "0 3 30 * *" # 03:30 UTC
```

Comment:
this is incorrect and will be `At 03:00 on day-of-month 30`, can we keep it as it was?